site/src/Librecores/ProjectRepoBundle/Entity/ProjectClassifier.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace Librecores\ProjectRepoBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * ProjectClassifier
+ *
+ * @ORM\Table(name="project_classifier")
+ * @ORM\Entity(repositoryClass="Librecores\ProjectRepoBundle\Repository\ProjectClassifierRepository")
+ */
+class ProjectClassifier
```

Comment:
This isn't a project classifier, which would be a tool or an application or an algorithm to classify a project. This is the classification that's assigned to a project. So something like "ProjectClassification" would be more appropriate.